### PR TITLE
Always build against upstream snapshot

### PR DESCRIPTION
**Pull request by iNikem**

### CODE REVIEW

File: custom/src/main/java/com/splunk/opentelemetry/micrometer/MicrometerBootstrapPackagesProvider.java [hunk 1]
```diff
@@ -17,17 +17,21 @@
...
-    return Arrays.asList(
-        // IMPORTANT: must be io.micrometer.core, because io.micrometer.signalfx needs to be in the
-        // agent classloader
-        "io.micrometer.core", "org.HdrHistogram", "org.LatencyUtils");
+  public void configure(Config config, BootstrapPackagesBuilder builder) {
+    builder.addAll(
+        Arrays.asList(
+            // IMPORTANT: must be io.micrometer.core, because io.micrometer.signalfx needs to be in
+            // the
+            // agent classloader
```
Comment (mateuszrzeszutek) [inline custom/src/main/java/com/splunk/opentelemetry/micrometer/MicrometerBootstrapPackagesProvider.java:34]:
```suggestion
            // the agent classloader
```

File: instrumentation/wildfly/src/main/java/com/splunk/opentelemetry/middleware/WildFlyAttributesInstrumentationModule.java [hunk 2]
```diff
@@ -69,12 +69,12 @@ public void transform(TypeTransformer typeTransformer) {
   public static class MiddlewareInitializedAdvice {
     @Advice.OnMethodEnter
     public static void onEnter() {
-      CallDepthThreadLocalMap.incrementCallDepth(ProductConfig.class);
+      CallDepth.forClass(ProductConfig.class).getAndIncrement();
```
Comment (mateuszrzeszutek) [inline instrumentation/wildfly/src/main/java/com/splunk/opentelemetry/middleware/WildFlyAttributesInstrumentationModule.java:72]: Should we pass `CallDepth` between enter/exit methods using a local variable?

**Review by mateuszrzeszutek — APPROVED** [review]

File: instrumentation/wildfly/src/main/java/com/splunk/opentelemetry/middleware/WildFlyAttributesInstrumentationModule.java (hunk 2 above)
Comment (iNikem) [reply instrumentation/wildfly/src/main/java/com/splunk/opentelemetry/middleware/WildFlyAttributesInstrumentationModule.java:72]: Sure. This PR or a follow up?

Comment (iNikem) [reply instrumentation/wildfly/src/main/java/com/splunk/opentelemetry/middleware/WildFlyAttributesInstrumentationModule.java:72]: I will do a separate PR after this one.

**Pull request merged**